docs/foundation/component-variables/README.md

```diff
@@ -69,19 +67,20 @@ Semantic naming
 
 #### Brand Consistency on More Platforms
   - Component variables are an export of the exact CSS styles used in the Cedar Vue components
-  - Ensures that core front-end components remain visually consistent across the co-op
+  - Ensures that core front-end components remain visually consistent across  REI sites
   - Third party teams can easily consume the component variables to integrate the Cedar design system into their architecture
 
 #### Ease of Maintainability
   - In the past, projects that could not consume Vue components had no way of using Cedar directly
   - Updates to the design system could not be pushed out to those projects unless they were implemented manually
+  (Comment: Can we rewrite these 2 points to specify why component variables can be easily maintained instead of just saying what was wrong in the past? Or, we can just put all 3 sentences in paragraph format, and not as bulleted list)
```

Comment:
i would rather just delete this entire "benefits" section i think, it feels more like a release announcement than something that belongs on the doc site long term

Reply:
Agree, but this section was reviewed by Pat in the Google docs, so I'll not play around much :) Let's just delete the first 2 bulleted item as they are irrelevant.